test/ppxlib-pp-ast/dune

```
@@ -0,0 +1,5 @@
+(cram
+ (package ppxlib-tools)
+ (deps
+  ../../bin/pp_ast.exe
```

```suggestion
%{bin:ppxlib-pp-ast}
```

I think we can change this -- if you do a `dune clean` and `dune runtest` directly then this won't pull in the right binary for the tests without explicitly depending on it like this.

---

Good catch! I'll give that a spin